graylog2-web-interface/src/views/logic/views/__snapshots__/CopyWidgetToDashboard.test.js.snap

```diff
@@ -17,27 +17,33 @@ Object {
         "546b13fe-eae7-4fb8-abb9-e0e3a907c3e6": Object {
           "col": 1,
           "height": 3,
-          "row": 5,
+          "row": NaN,
```

Comment:
Isn't this an indication that the code has an issue? IMO it should not generate a `NaN` value for the `row`.